docs/tutorials/orqviz.rst

```diff
@@ -0,0 +1,66 @@
+======
+Orqviz
+======
+
+To aid in the development of variational quantum algorithms, we have developed a visualization tool called OrqViz. It is a Python package that allows users to visualize the cost function landscape of a variational quantum algorithm. It is designed to be used in Jupyter notebooks, and can be installed via pip.
```

Comment:
`orqviz` is actually all lowercase :)

Reply:
I think it's also worth adding that it can visualize any cost function, it doesn't even need to be quantum.